.gitignore

```diff
@@ -35,4 +35,12 @@ packages/
 nuget.exe
 
 .idea/
-*.iml
\ No newline at end of file
+*.iml
+
+BenchmarkDotNet.IntegrationTests/**/Program.cs
+BenchmarkDotNet.IntegrationTests/**/Program.csproj
+BenchmarkDotNet.IntegrationTests/*.csv
+BenchmarkDotNet.IntegrationTests/*.txt
+BenchmarkDotNet.IntegrationTests/*.md
+BenchmarkDotNet.IntegrationTests/*.R
+BenchmarkDotNet.Autogenerated/*
```

Comment:
@khellang Thanks! It used to be a best practice (at least in corefxlab repo for some time). It is good to know it is an anti-pattern now ;)